zds/utils/templatetags/nb_word_to_time.py

```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+
+from django import template
+from zds.settings import ZDS_APP
+register = template.Library()
+
+
+@register.filter(name="nb_word_to_icontime")
+def nb_word_to_icontime(nb_word):
+    """
+    Gets the icon for this time to read the content
+    :param nb_word:
+    :return:
+    """
+    association = ["icon-15-min", "icon-30-min", "icon-45-min", "icon-60-min", "icon-more-min"]
```

Comment:
façon, l'échelle ne va pas de base car tel que c'est fait 16 minutes te dira "30".

Par contre, faut voir, plus on augmente le nombre de pas, plus il faudra que le front/webdesign suive. 

Et pour ce qui est du logarithme ça va juste complexifier le code, mais why not.